### DISCUSSION TITLE
绯月ScarletMoon 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#196] [模板讨论]
绯月ScarletMoon 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by hecarli555** [comment]
提示失败，，，
Failed at 3/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://bbs.kfpromax.com/kf_growup.php

**Reply by wjf0214** [reply]
感觉是登录失败了。而且还是我登录验证没判断到的提示情况。。可以单独请求一下第一条看看登录失败提示的什么。

**Reply by wjf0214** [reply]
是不是账号输入错了，提示账号不存在，目前我测试应该可能就这种情况了。

**Reply by hecarli555** [reply]
我错了大佬。。。是我的问题，是我转跳了2dfan的站，注册的2dfan的号，，，，

**Comment by booskt** [comment]
Failed at 2/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 302: Found,
Request URL: https://bbs.kfpromax.com/index.php

**Reply by wjf0214** [reply]
302应该是登录失败了。第一条请求看看是返回了什么。

**Reply by booskt** [reply]
Status Code: 200 
请求成功条件断言 支持正则，任意条件满足即为请求成功（没有失败条件命中时） 

status
200
=
200 
请求失败条件断言 支持正则，任意条件满足即为请求失败 

content
密码错误或安全问题错误,您还可以尝试 [\d+] 次
=

content
用户[A-Za-z0-9]+不存在
=

**Reply by booskt** [reply]
用户名是中文

**Reply by wjf0214** [reply]
再试试，修复了中文账户登录的问题。

**Reply by booskt** [reply]
Failed at 2/8 request,
Fail assert: {"re": "用户[\\s\\S]+不存在", "from": "content"} from failed_asserts,
Request URL: https://bbs.kfpromax.com/login.php

**Reply by wjf0214** [reply]
更新了，再试试吧，之前保存的时候框架前端自动加了urlencode 导致失败了。刚修复了。

**Reply by booskt** [reply]
感谢！已经成功了
